hdl_localization_node:
  ros__parameters:
    # Topic settings
    points_topic: /livox/lidar
    imu_topic: /livox/imu
    
    # Frame IDs
    odom_child_frame_id: base_link
    robot_odom_frame_id: odom
    
    # IMU settings
    use_imu: false
    invert_acc: false
    invert_gyro: false
    
    # Localization settings
    use_global_localization: true
    enable_robot_odometry_prediction: false
    
    # Algorithm parameters
    reg_method: GICP_PCL
    ndt_neighbor_search_method: DIRECT7
    ndt_neighbor_search_radius: 2.0
    ndt_resolution: 1.0
    downsample_resolution: 0.25
    cool_time_duration: 1.0
    
    # Initial pose
    specify_init_pose: true
    init_pos_x: 0.0
    init_pos_y: 0.0
    init_pos_z: 0.0
    init_ori_w: 1.0
    init_ori_x: 0.0
    init_ori_y: 0.0
    init_ori_z: 0.0
    
    # Visualization
    plot_estimation_errors: false

globalmap_server_node:
  ros__parameters:
    globalmap_pcd: /home/rtc/ros2_ws/map/mapmoi/mapmoi.ply
    convert_utm_to_local: true
    downsample_resolution: 0.1